secretmanager/create_secret_with_user_managed_replication.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package secretmanager
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	secretmanager "cloud.google.com/go/secretmanager/apiv1"
+	secretmanagerpb "google.golang.org/genproto/googleapis/cloud/secretmanager/v1"
+)
+
+// createUmmrSecret creates a new secret with the given name with user managed replication.
+// A secret is a logical wrapper around a collection of secret versions. Secret
+// versions hold the actual secret material.
+func createUmmrSecret(w io.Writer, parent, id string, locations []string) error {
```

Comment:
nit: createUserManagedReplicaSecret reads just as well. Is Ummr a common abbreviation?

Reply:
Ummr is common internally. updated to use createUserManagedReplicationSecret